Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -21,7 +22,7 @@
 from Configuration.Eras.Modifier_run2_jme_2017_cff import run2_jme_2017
 from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
 
-Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2016,pixel_2016,run2_jme_2016]),
+Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016, run2_ECAL_2016, run2_egamma_2016,pixel_2016,run2_jme_2016]),
```

Comment:
is run2_ECAL_2016 and run2_egamma_2016 now actually different from the run2_ECAL_2017 and run2_egamma_2017, respectively?
Do we really still need different modifiers?

Reply:
I think for run2_ECAL_X, it is more for ECAL (Amina and Fabrice - cant find them here) to answer. But I see that run2_ECAL_2017 is used here as well:https://cmssdt.cern.ch/lxr/source/RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py  For run2_egamma_X, we could do that (2017 and 2018 are the same, just need to integrate 2016 as well) now that 2016 to 2018 is made uniform unless @Sam-Harper has any other suggestions?

Reply:
the particleFlowZeroSuppressionECAL I think is now fully redundant due to global tag pf rec-hit thresholds so we dont care about it any more.  Also it was in hindsight a mistake to do in the first place.

as the run2_ECAL_2016 etc. I think we made our bed and we should lie in.  I absolutely object to using run2_X_2017 for 2016 data and I dont think we should be removing modifiers between releases in the same major version.  Ie given the run2_Ecal/Egamma_2017 exists in CMSSW_10_6_X, it should also exist in CMSSW_10_6_>X

I'm fine for merging them in 11_X though. I would prefer that to be a separate PR on top of this one otherwise we have different PR for 10_6 and 11_0